project/src/components/Requests.js

```diff
@@ -18,18 +19,96 @@ module.exports = class SendRequest {
         });
     }
 
-    static sendForLogIn(url, method, data, successFunc, btnGroup, props, toggleClasses) {
+    static sendForLogIn(url, method, data, successFunc, btnGroup, props, toggleClasses, user, saveToLocalStorage) {
         $.ajax({
             url,
             method,
             data,
             contentType: 'application/json; charset=utf-8',
             success: function (newData) {
-                successFunc(newData, btnGroup, props, toggleClasses)
+                successFunc(newData, btnGroup, props, toggleClasses, user, saveToLocalStorage)
             },
             error: ((data) => {
                 console.log(data.status + ': ' + data.statusText);
             })
         });
     }
+
+    static sendForFriends(url, method, user, props){
```

Comment:
These functions are absolutely the same.
Copy-paste detected.
Have to be fixed in the firstly!